src/integrators/gpt/gpt.cpp

```diff
@@ -1188,6 +1188,10 @@ class GradientPathTracer {
 GradientPathIntegrator::GradientPathIntegrator(const Properties &props)
 	: MonteCarloIntegrator(props)
 {
+	m_config.m_maxDepth = props.getInteger("maxDepth", -1);
+	m_config.m_minDepth = props.getInteger("minDepth", -1);
+	m_config.m_rrDepth = props.getInteger("rrDepth", 5);
+	m_config.m_strictNormals = props.getBoolean("strictNormals", false);

```

Comment:
Apparently these should be set automatically in the superclass constructor, and also serialized by the superclass...